packages-native/popup-menu/src/PopupMenu.xml

```diff
@@ -3,14 +3,14 @@
         supportedPlatform="Native"
         xmlns="http://www.mendix.com/widget/1.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../../../node_modules/mendix/custom_widget.xsd">
-    <name>Popup Menu</name>
+    <name>Popup menu</name>
```

Comment:
Should this be also Pop-up menu?

Reply:
If yes, then it should be fixed along the whole file

Reply:
For example ?